livekit-plugins/livekit-plugins-deepgram/livekit/plugins/deepgram/stt.py

```diff
@@ -498,6 +498,14 @@ async def recv_task(ws: aiohttp.ClientWebSocketResponse):
                         [asyncio.gather(*tasks), wait_reconnect_task],
                         return_when=asyncio.FIRST_COMPLETED,
                     )  # type: ignore
+
+                    # need to propagate exceptions from completed tasks
+                    for task in done:
+                        if task != wait_reconnect_task:
+                            exc = task.exception()
+                            if exc is not None:
+                                raise exc
```

Comment:
can be simplified to:
```suggestion
                            task.result()
```